packet/arp.go

```diff
@@ -67,29 +69,40 @@ func InitEmptyARPPacket(packet *Packet) bool {
 	return true
 }
 
-// InitARPRequestPacket initialize ARP request packet for IPv4
-// protocol request with broadcast (zero) for THA (Target HW
-// address). SHA and SPA specify sender MAC and IP addresses, TPA
-// specifies IP address for host which request is sent
-// for. Destionation MAC address is L2 Ethernet header is set to
-// FF:FF:FF:FF:FF:FF (broadcast) and source address is set to SHA.
-func InitARPRequestPacket(packet *Packet, SHA [common.EtherAddrLen]uint8, SPA, TPA uint32) bool {
+// InitARPCommonData allocates ARP packet, fills ether header and
+// arp hrd, pro, hln, pln with values for ether and IPv4
+func InitARPCommonData(packet *Packet) bool {
```

Comment:
Let's make this function private to this file, i.e. from lower case letter. It is not needed from outside.